great_ape_safe/ape_api/ebtc.py

```diff
@@ -295,6 +301,20 @@ class governanceRoles(Enum):
                     "signature": self.governance_signatures["SET_FALLBACK_CALLER_SIG"],
                 },
             ],
+            governanceRoles.STETH_MARKET_RATE_SWITCHER.value: [
```

Comment:
targets and signatures matches with its own outstanding [pr](https://github.com/ebtc-protocol/ebtc/pull/784) ✅